Improve Schema loader document (#368)
scalar-labs/scalardb: schema-loader/README.md

# Scalar DB Schema Loader

> This tool creates/deletes Scalar DB schemas (namespaces and tables) based on a provided schema file.

Reviewer: ```suggestion
Scalar DB Schema Loader creates and deletes Scalar DB schemas (namespaces and tables) on the basis of a provided schema file.
```

> Also, it automatically adds the Scalar DB transaction metadata (used in the Consensus Commit protocol) to the tables when you set the `transaction` parameter `true` in the schema file.

Reviewer: ```suggestion
Also, it automatically adds the Scalar DB transaction metadata (used in the Consensus Commit protocol) to the tables when you set the `transaction` parameter to `true` in the schema file.
```

There are two ways to specify general CLI options in schema-loader:

>   - Pass a Scalar DB configuration file and database-specific options additionally.

Reviewer: ```suggestion
  - Pass a Scalar DB configuration file and database/storage-specific options additionally.
```

  - Pass the options without a Scalar DB configuration.

Note that this tool supports only basic options to create/delete a table.

> If you want to use advanced feature of the database, please alter your table after creating it with this tool.

Reviewer: ```suggestion
If you want to use advanced features of the database, please alter your table after creating it with this tool.
```
since I see `storage-spacific` in the other places in the doc.

# Usage

## Install

The release versions of `schema-loader` can be downloaded from [releases](https://github.com/scalar-labs/scalardb/releases) page of Scalar DB

## Build

In case you want to build `schema-loader` from the source:
```console
$ ./gradlew schema-loader:shadowJar
```
- The built fat jar file is `schema-loader/build/libs/scalardb-schema-loader-<version>.jar`

## Docker

You can pull the docker image from [Scalar's container registry](https://github.com/orgs/scalar-labs/packages/container/package/scalardb-schema-loader).
```console
docker run --rm -v <your_local_schema_file_path>:<schema_file_path_in_docker> [-v <your_local_config_file_path>:<config_file_path_in_docker>] ghcr.io/scalar-labs/scalardb-schema-loader:<version> <command_arguments>
```

> - You can specify the same command arguments as bellow by simply replacing `java -jar scalardb-schema-loader-<version>.jar` with `docker run --rm -v <your_local_schema_file_path>:<schema_file_path_in_docker> [-v <your_local_config_file_path>:<config_file_path_in_docker>] ghcr.io/scalar-labs/scalardb-schema-loader:<version>`

Reviewer: ```suggestion
- Note that you can specify the same command arguments even if you use the fat jar or the container. The example commands in the next section are shown with a jar, but you can run the commands with the container in the same way by replacing `java -jar scalardb-schema-loader-<version>.jar` with `docker run --rm -v <your_local_schema_file_path>:<schema_file_path_in_docker> [-v <your_local_config_file_path>:<config_file_path_in_docker>] ghcr.io/scalar-labs/scalardb-schema-loader:<version>`.
```

  }
}
```

> You can also specify storage-specific options in the table definition as follows:

Reviewer: ```suggestion
You can also specify database/storage-specific options in the table definition as follows:
```
since we see `database-specific` in other places in the doc.

```json
{
  "sample_db.sample_table3": {
    "partition-key": [
      "c1"
    ],
    "columns": {
      "c1": "INT",
      "c2": "TEXT",
      "c3": "BLOB"
    },
    "compaction-strategy": "LCS",
    "ru": 5000
  }
}
```

> The storage-specific options you can specify are as follows:

Reviewer: ```suggestion
The database/storage-specific options you can specify are as follows:
```
